### PR TITLE
Feature/kamraci enchancements

### PR DESCRIPTION
**Pull request by jakubkrzysztofsikora**
zrzut ekranu 2019-02-8 o 19 16 07

Added filters for friend event subpage

### CODE REVIEW

File: src/components/EventFilters.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import SingleFilter from './SingleFilter';
+
+export default ({ containerClassName, singleFilterClassName, filters }) => {
```
Comment (BartlomiejGlac) [inline src/components/EventFilters.jsx]: Are we adding proptypes?

File: src/constants/eventLocations.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default {
+    kato: 'Katowice',
```
Comment (BartlomiejGlac) [inline src/constants/eventLocations.js]: katowice maybe :)

Comment (jakubkrzysztofsikora) [reply src/constants/eventLocations.js]: Is there any other `kato` we can be confused with? :D This is a nice name. :)

File: src/components/EventFilters.jsx (hunk 1 above)
Comment (jakubkrzysztofsikora) [reply src/components/EventFilters.jsx]: We don't have it now, I think adding it now, requires refactor of the rest of the components. Not a part of this scope.

**Pull request merged**